packages/suite/src/components/wallet/TransactionItem/TransactionItem.tsx

```diff
@@ -238,242 +217,268 @@ export const TransactionItem = memo(
             <Wrapper
```

Comment:
offtopic: This whole component is crazy, really hard to review and should be divided into multiple components.

Reply:
Agree. But I didn't want to refactor this, as it's gonna change anyway.